### ISSUE TITLE
Showing ALL instagram beta videos instead of only the "approved" ones

### ISSUE THREAD

**Issue by marcodejongh** [lemeryfertitta/Climbdex#78]

**Comment by marcodejongh** [comment]
Oops pressed enter too soon, updating to add a description

**Comment by lemeryfertitta** [comment]
Seems like a nice addition, you're welcome to take a stab. I vaguely remember trying to play around with the instagram API to improve the beta page (i.e. to fix https://github.com/lemeryfertitta/Climbdex/issues/27) but didn't have much luck on initial investigation.

**Comment by gardaholm** [comment]
would be nice addition, don't know how often aurora updates the list, but by now none of my instagram videos have been added. 

I just realized that the aurora app (tension board at least) shows different caption examples, depending on the device you are using. On Android I get a different caption suggestion than on iOS. Maybe this is an issue for them?

**Comment by marcodejongh** [comment]
Yeah I think the real problem is that it seems like IG only allows tag searches through their API.
And when tagging beta videos, there's no easy to tie back to a climb tag added, making it much harder as it means  human will have to read the description and or watch the video.
Ofc the description could be regex'd but still not ideal.

If every climb just had an id, and beta videos had #kilterhw-$id in their tags, it would be much more efficient to pull in the new beta videos
